Formula/gauge.rb

```diff
@@ -42,6 +42,12 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
```

Comment:
@ilovezfs - right, we will do that in the next release (requires change to gauge). In the meantime, would adding this to the `desc` in the formula be acceptable? So that at least `brew info gauge` can highlight this change?

apologies for the back and forth, and thank you for the dedication.